package.json

```diff
@@ -1,7 +1,7 @@
 {
   "name": "@replit/codemirror-css-color-picker",
   "description": "Enables a color picker input next to css colors",
-  "version": "0.19.6",
+  "version": "0.20.0",
```

Comment:
unfortunately it’s [not backwards compatible](https://github.com/replit/Codemirror-CSS-color-picker/blob/e0ecb5ec2a8f947f9e68aeccc9cf10434d9793fc/CHANGELOG.md)

Reply:
In that case we should do a major version bump to follow semantic version, or we just bump a patch and stick to codemirror's version since we're on v0 and https://semver.org/#spec-item-4

Reply:
huh I didn’t know you can bump patch versions in 0.x, I’ll do that